README.md

```diff
@@ -1,8 +1,87 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Homa Almasieh . Jarvis Consulting
 
+ I recently completed Data Analytics BootCamp from the University of Toronto focused on gaining technical programming skills in Excel, VBA, Python, R, JavaScript, SQL Databases, Tableau, Big Data, and Machine Learning. I worked with various datasets during the BootCamp program and according to the content of the projects, I used various tools to analyze the data, design the model and visualize the finding through Tableau and dashboards. In addition, I am a former Assistant Professor of University in the area of Applied Mathematics with a brilliant and valuable background in the field of teaching and scientific research. With this extensive background of Mathematics and Data, I started  my training program at Jarvis by developing various projects and this mentorship helped me to gain skills in increasing code efficiency, debugging, and gaining comprehensive knowledge in Linux/Bash, Git, Java, Docker, RDBMS and Springboot. I am a self-initiated and avid learner with high attention to detail, critical thinking and logical problem-solving. My adaptability allows me to work in many different environments, in addition to speaking with people from various backgrounds. I am looking forward to speaking about my academic and professional background for developing positions mainly in the data analytics field.
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, Machine Learning and Data Analytics, Statistics, RDBMS/SQL, Tableau, Agile/Scrum, Git
+
+**Competent:** Pandas, Numpy, Scipy, Scikit-Learn, DHTML/CSS
+
+**Familiar:** R, APIs, Tensorflow/Deep learning, Flask, Google Cloud Platform(GCP)
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_HomaAlmasieh](https://github.com/jarviscanada/jarvis_data_eng_HomaAlmasieh)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HomaAlmasieh/tree/master/linux_sql)]: Implemented a Monitoring Agent on a node/server within the linux cluster to collect the hardware specifications and monitor the cluster node resource usage in real time. This Monitoring Agent contains a PostgreSQL server installed within a Docker container that collects information submitted through bash scripts installed on each node.
```

Comment:
Implemented a Monitoring Agent on a node/server within the Linux cluster to collect the hardware specifications and monitor the cluster node resource usage in real-time. This Monitoring Agent contains a PostgreSQL server installed within a Docker container that collects information submitted through bash scripts installed on each node.

fixed typo

Reply:
missing keywords
https://github.com/jarviscanada/jarvis_profile_builder/blob/master/profile.yaml
![image](https://user-images.githubusercontent.com/58951265/136441327-af3de6b8-8dfd-473a-8da4-c0e63de8b8b0.png)

Reply:
I have added the keywords to the project description.